conf/app.routes

```diff
@@ -41,6 +41,9 @@ POST        /form/business-partners/partner-national-insurance-number   @uk.gov.
 GET         /form/business-partners/choose-address                      @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersChooseAddressController.load()
 POST        /form/business-partners/choose-address                      @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnersChooseAddressController.next()
 
+GET         /form/business-partners/enter-partners-address              @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnerEnterAddressController.load()
+POST        /form/business-partners/enter-partners-address              @uk.gov.hmrc.fhregistrationfrontend.controllers.BusinessPartnerEnterAddressController.next()
+
```

Comment:
A minor point but this should be `enter-partner-address` rather than `enter-partners-address` - without the 's' on partner - to match the URL on the ticket and keep it consistent with the other new page URLs.